src/components/Views/TabBarNavigation/index.js

```diff
@@ -3,9 +3,15 @@ import { connect } from 'react-redux';
 import { withNavigationFocus } from 'react-navigation';
 
 import TabBarNavigation from './TabBarNavigation';
+import { drawerClose, drawerOpen } from '../../../ducks/drawers';
 
 const mapStateToProps = state => ({
-    showSavedModal: state.app.modals.saved
-  });
+  showSavedModal: state.app.modals.saved
+});
 
-export default connect(mapStateToProps)(TabBarNavigation);
+const mapDispatchToProps = dispatch => ({
+  onDrawerClose: () => dispatch(drawerClose('opened')),
```

Comment:
let's call this 'observations' instead of 'opened'

Reply:
👍